homeassistant/components/linode/switch.py

```diff
@@ -51,24 +51,12 @@ def __init__(self, li, node_id):  # pylint: disable=invalid-name
         self._linode = li
         self._node_id = node_id
         self.data = None
-        self._state = None
-        self._attrs = {}
-        self._name = None
-
-    @property
-    def name(self):
-        """Return the name of the switch."""
-        return self._name
+        self._attr_extra_state_attributes = {}
 
     @property
     def is_on(self):
         """Return true if switch is on."""
-        return self._state
-
-    @property
-    def extra_state_attributes(self):
-        """Return the state attributes of the Linode Node."""
-        return self._attrs
+        return self._attr_is_on
```

Comment:
This is already the default for the `is_on` property. The whole property method can be removed.

Reply:
Doh! should be ok now.